src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts

```diff
@@ -16,7 +16,6 @@ import {
 } from "../../../../../data/entity_registry";
 import { HELPERS_CRUD } from "../../../../../data/helpers_crud";
 import { showConfirmationDialog } from "../../../../../dialogs/generic/show-dialog-box";
-import { hideMoreInfoDialog } from "../../../../../dialogs/more-info/show-ha-more-info-dialog";
```

Comment:
Can we remove this function? I think it is not longer needed?

Reply:
It's used in ha-config-entities but I'm not sure why :

```ts
  public disconnectedCallback() {
    super.disconnectedCallback();
    hideMoreInfoDialog(this);
  }
```

Reply:
It used to be 2 different dialogs, but they no longer are

Reply:
Ok I will remove it 🙂